style.css

```diff
@@ -11,24 +11,45 @@
 
 body {
   /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+	/* lets see what Arial looks like*/
+  font-family: Arial;
+background-color: #afeeee;
 }
 
+
+h1 {
+
+color: #680000;
+background-color: #003362
+
+
 /*
  * Center content by setting a width but giving the left and right
  * margin a value of auto
  */
 .container {
-  width: 700px;
+  width: 550px;
   margin: 0 auto;
-}
+
 
 .content ul {
-  margin: 0;
-  padding: 0;
+  margin: 20;
```

Comment:
Although this may work, you should always specify the unit of measurement:

- `5px` would be 5 screen pixels
- `5em` means 5 times the containing elements font size
- `5rem` indicates 5 times the font size of the main html document (not the containing element but the root of the html itself)
- `5%` relative to the containing elements width

There are probably more. I tend to recommend using `px` when learning and then looking into both `rem` and `em`